internal/operands/common-templates/reconcile.go

```diff
@@ -54,6 +60,34 @@ func (c *commonTemplates) WatchTypes() []runtime.Object {
 	return nil
 }
 
+func (c *commonTemplates) PauseCRs(request *common.Request) error {
+	patch := []byte(`{"metadata":{"annotations":{"kubevirt.io/operator.paused": "true"}}}`)
+	var kubevirtCommonTemplatesBundles kvsspv1.KubevirtCommonTemplatesBundleList
+	err := request.Client.List(request.Context, &kubevirtCommonTemplatesBundles, &client.ListOptions{})
+	if err != nil {
+		if errors.IsNotFound(err) {
+			request.Logger.Info(fmt.Sprintf("No legacy common template bundle CR found"))
+			return nil
+		} else {
+			request.Logger.Error(err, fmt.Sprintf("Error listing common template bundle CRs: %s", err))
+			return err
+		}
+	}
+	for _, kubevirtCommonTemplatesBundle := range kubevirtCommonTemplatesBundles.Items {
+		err = request.Client.Patch(request.Context, &kubevirtCommonTemplatesBundle,
```

Comment:
Can you add a check if the annotation is already present? And only call patch if it is not.